bundles/inheritance.rst

```diff
@@ -93,8 +93,8 @@ The same goes for routing files and some other resources.
 
     The overriding of resources only works when you refer to resources with
     the ``@FOSUserBundle/Resources/config/routing/security.xml`` method.
-    If you refer to resources without using the ``@BundleName`` shortcut, they
-    can't be overridden in this way.
+    You need to use the ``@BundleName`` shortcut when refering to resources
+    so they can be successfully overridden.
```

Comment:
There has been confusion in the past about which resources we refer to here. I think we should make more clear that this note does not apply when we talk about templates as the overriding of them works no matter if you use the bundle syntax or the native Twig namespace syntax (I know that the syntax is not exactly the same as what we use to refer to other resources, but that's not so obvious).